test/test_c10d.py

```diff
@@ -3171,9 +3171,12 @@ def _test_nccl_errors_blocking(self, func):
         process_group.allreduce(torch.rand(10).cuda(self.rank))
         if self.rank == 0:
             work = process_group.allreduce(torch.rand(10).cuda(self.rank))
-            with self.assertRaises(RuntimeError):
+            try:
                 # Operation would time out in blocking mode.
                 work.wait()
+            except Exception as e:
+                self.assertEqual("Operation timed out!", str(e))
+                a = torch.rand(10).cuda(self.rank)
```

Comment:
what is the purpose for this? are you verifying GPU is not stuck at 100% and memory copy is still work. Could you add comments for this?

Reply:
yes, that  is the main purpose, will add comment